docs/components.md

```diff
@@ -365,8 +365,14 @@ var Login = {
 	login: function() {/*...*/},
 	view: function() {
 		return m(".login", [
-			m("input[type=text]", {oninput: m.withAttr("value", this.setUsername.bind(this)), value: this.username}),
-			m("input[type=password]", {oninput: m.withAttr("value", this.setPassword.bind(this)), value: this.password}),
```

Comment:
Good grief, what a mess this was. A large part of the beauty of closures by default will be removing all these horrid custom component *methods* and `this` references.